packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -30,12 +30,12 @@ import { BlockListBlockContext } from './block';
 import ELEMENTS from './block-wrapper-elements';
 
 /**
- * This hook is used to lighly mark an element as a block element. Call this
- * hook and pass the returned props to the element to mark as a block. If you
- * define a ref for the element, it is important to pass the ref to this hook,
- * which the hooks in turn will pass to the component through the props it
- * returns. Optionally, you can also pass any other props through this hook, and
- * they will be merged and returned.
+ * This hook is used to lightly mark an element as a block element. The element
+ * should be the outer element of a block. Call this hook and pass the returned
```

Comment:
* _outer_ -> _outermost_?
* maybe also mention that only one top-level element may be returned by the component (i.e., no Fragments, no arrays)

Reply:
What doe you mean? We allow fragments for the block controls, right?